Can't run from commandline

Tests pass--which include running `Seqr.main()` without error--but I get an exception if I try running directly from the commandline. This suggests to me that there is a problem with the `pom` file.
I tried removing `<scope> test </scope>` [here](https://github.com/DCGenomics/seqr/blob/26a09883e858a8fec2a225a558302ee4489c392c/pom.xml#L55-L61) but that did not help.

Forgot to include the error output:

``` java
bash-4.1$ java -jar   target/seqr-4.10.4-SNAPSHOT-jar-with-dependencies.jar   search testdata/data/test.fasta --db testdata/solr/
log4j:WARN No appenders could be found for logger (org.apache.solr.core.SolrResourceLoader).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
Exception in thread "main" java.util.ServiceConfigurationError: Cannot instantiate SPI class: org.apache.lucene.codecs.asserting.AssertingCodec
    at org.apache.lucene.util.NamedSPILoader.reload(NamedSPILoader.java:77)
    at org.apache.lucene.util.NamedSPILoader.<init>(NamedSPILoader.java:47)
    at org.apache.lucene.util.NamedSPILoader.<init>(NamedSPILoader.java:37)
    at org.apache.lucene.codecs.Codec.<clinit>(Codec.java:41)
    at org.apache.solr.core.SolrResourceLoader.reloadLuceneSPI(SolrResourceLoader.java:206)
    at org.apache.solr.core.SolrResourceLoader.<init>(SolrResourceLoader.java:142)
    at org.apache.solr.core.SolrResourceLoader.<init>(SolrResourceLoader.java:256)
    at org.apache.solr.core.CoreContainer.<init>(CoreContainer.java:134)
    at gov.nih.nlm.ncbi.seqr.Seqr.handleCommand(Seqr.java:204)
    at gov.nih.nlm.ncbi.seqr.Seqr.main(Seqr.java:67)
Caused by: java.lang.IllegalArgumentException: An SPI class of type org.apache.lucene.codecs.PostingsFormat with name 'Lucene41' does not exist.  You need to add the corresponding JAR file supporting this SPI to your classpath.  The current classpath supports the following names: [MockFixedIntBlock, MockVariableIntBlock, MockRandom, MockSep, NestedPulsing, RAMOnly, Lucene41WithOrds, TestBloomFilteredLucene41Postings, Asserting, Lucene40]
    at org.apache.lucene.util.NamedSPILoader.lookup(NamedSPILoader.java:109)
    at org.apache.lucene.codecs.PostingsFormat.forName(PostingsFormat.java:100)
    at org.apache.lucene.codecs.lucene410.Lucene410Codec.<init>(Lucene410Codec.java:131)
    at org.apache.lucene.codecs.asserting.AssertingCodec.<init>(AssertingCodec.java:40)
    at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
    at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
    at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
    at java.lang.Class.newInstance(Class.java:442)
    at org.apache.lucene.util.NamedSPILoader.reload(NamedSPILoader.java:67)
    ... 9 more
```

--- a/pom.xml
+++ b/pom.xml
@@ -2,7 +2,53 @@
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"> 
  <build>
   <plugins>
-   <plugin>
+
+
+
+
+      <!-- Shade / Self-contained JAR -->
+      <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-shade-plugin</artifactId>
+          <!-- <version>2.0</version> -->
+          <version>2.2</version>
+          <executions>
+              <execution>
+                  <phase>package</phase>
+                  <goals>
+                      <goal>shade</goal>
+                  </goals>
+                  <configuration>
+                      <!-- <minimizeJar>true</minimizeJar> -->
+                      <minimizeJar>false</minimizeJar>
+                      <transformers>
+                          <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
+                          <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                              <mainClass>gov.nih.nlm.ncbi.seqr.Seqr</mainClass>
+                          </transformer>
+                      </transformers>
+
+                      <!-- fix SecurityException: Invalid signature file digest for Manifest main attributes -->
+                      <filters>
+                          <filter>
+                              <artifact>*:*</artifact>
+                              <excludes>
+                                  <exclude>META-INF/*.SF</exclude>
+                                  <exclude>META-INF/*.DSA</exclude>
+                                  <exclude>META-INF/*.RSA</exclude>
+                              </excludes>
+                          </filter>
+                      </filters>
+
+                  </configuration>
+              </execution>
+          </executions>
+      </plugin>
+
+
+
+
+<!--   <plugin>
      <artifactId>maven-assembly-plugin</artifactId>
      <configuration>
        <archive>
@@ -23,7 +69,7 @@
          </goals>
        </execution>
      </executions>
-   </plugin>
+   </plugin> -->
   </plugins>
  </build>
     <modelVersion>4.0.0</modelVersion>
@@ -52,13 +98,11 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <version>4.12</version>
-            <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.lucene</groupId>
             <artifactId>lucene-test-framework</artifactId>
             <version>${solr.version}</version>
-            <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.lucene</groupId>
@@ -69,7 +113,6 @@
             <groupId>org.apache.solr</groupId>
             <artifactId>solr-test-framework</artifactId>
             <version>${solr.version}</version>
-            <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>

